kernel/kernel-api/src/test/scala/io/delta/kernel/internal/util/FileNamesSuite.scala

```diff
@@ -59,4 +105,33 @@ class FileNamesSuite extends AnyFunSuite {
       new Path("/a/00000000000000000001.checkpoint.0000000004.0000000005.parquet"),
       new Path("/a/00000000000000000001.checkpoint.0000000005.0000000005.parquet")))
   }
+
+  ///////////////////////////////////
+  // Is <type> file checkers tests //
+  ///////////////////////////////////
+
+  test("is checkpoint file") {
```

Comment:
should we check that the specific checkpoint types don't match the others?
i.e. `assert(!isClassicCheckpointFile(checkpointMultiPart)`?

Reply:
sure!